codespell_lib/data/dictionary.txt

```diff
@@ -7261,6 +7295,7 @@ habeus->habeas
 Habsbourg->Habsburg
 hace->have
 hach->hatch, hack, hash,
+haders->headers
```

Comment:
Can also be `haters` :stuck_out_tongue_winking_eye:

Reply:
Done